sentences/de/fan_HassTurnOff.yaml

```
@@ -5,6 +5,7 @@ intents:
       - sentences:
           - "(schalte | stoppe) (den Ventilator | (die | alle) Ventilatoren) <area> [<turn_off>]"
           - "(schalte | stoppe) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_off>]"
+          - "(mach | mache) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_off>]"
```

Similar to the existing sentences above:
```suggestion
          - "(mach | mache) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_off>]"
          - "(mach | mache) (den Ventilator | (die | alle) Ventilatoren) <area> [<turn_off>]"
```